client-cli/src/test/java/org/eclipse/dataspaceconnector/identityhub/cli/VerifiableCredentialsCommandTest.java

```diff
@@ -151,11 +171,15 @@ void add_invalidPrivateKey_fails() throws JsonProcessingException {
         var errContent = err.toString();
 
         // assert
-        assertThat(exitCode).isNotEqualTo(0);
+        assertThat(exitCode).isNotZero();
         assertThat(outContent).isEmpty();
         assertThat(errContent).contains("Error while signing Verifiable Credential");
     }
 
+    private int executeGetSelfDescription() {
+        return cmd.execute("-s", HUB_URL, "sd");
```

Comment:
Consider adding a suboption for the command like "sd get". I assume we will add "sd upload" in the future.

Reply:
Done in https://github.com/agera-edc/IdentityHub/pull/37/commits/14d238e422c0e272ec643caa5aaa69820a6f4955